//-----------------------------------------------------------------------------
// {C} Copyright 2019 Pensando Systems Inc. All rights reserved
//-----------------------------------------------------------------------------
#include "nic/hal/hal.hpp"

namespace hal {
namespace aclqos {

hal_ret_t hal_acl_config_init(void);

}
}
